pkg/kubelet/dockershim/BUILD

```diff
@@ -78,6 +78,7 @@ go_library(
         "@io_bazel_rules_go//go/platform:windows": [
             "//pkg/kubelet/apis:go_default_library",
             "//pkg/kubelet/winstats:go_default_library",
+            "//vendor/github.com/docker/docker/pkg/sysinfo:go_default_library",
```

Comment:
You need to add `cpu_limits.go` to the list starting on line 6 or else it won't be compiled

Reply:
@benmoss - do you mean in test/e2e/BUILD?

Reply:
ran hack/update-bazel.sh which should resolve it